tests/gold_tests/cache/disjoint-wait-for-cache.test.py

```diff
@@ -22,8 +22,10 @@
 Same as cache-generaertaion-disjoint, but uses proxy.config.http.wait_for_cache which should delay
 the server from accepting connection till the cache is loaded
 '''
-# need Curl
-Test.SkipIf(Condition.true("This test fails at the moment as is turned off"))
+
+Test.SkipUnless(
```

Comment:
This should be removed. @zwoop removed these checks in 6c1a90f4f209a02cf1345ac4c20e2e67c4c58db1

Reply:
done